samples/openapi3/client/petstore/python-nextgen-aiohttp/petstore_api/models/format_test.py

```diff
@@ -119,8 +113,8 @@ def from_dict(cls, obj: dict) -> FormatTest:
             "decimal": obj.get("decimal"),
             "string": obj.get("string"),
             "string_with_double_quote_pattern": obj.get("string_with_double_quote_pattern"),
-            "byte": bytearray.from_dict(obj.get("byte")) if obj.get("byte") is not None else None,
-            "binary": bytearray.from_dict(obj.get("binary")) if obj.get("binary") is not None else None,
+            "byte": obj.get("byte"),
```

Comment:
byte is base64 encoded characters. How will pydantic store that client side and send it? How about adding tests of:
- client side instantiation
- serialization
- deserialization

Reply:
This PR is a follow-up PR to https://github.com/OpenAPITools/openapi-generator/pull/15076, which uses `bytearray` for binary response/type and there's a test.

(whether it's byte or binary format, such thing has no `from_dict` method)